components/src/core/user-menu/user-menu.tsx

```diff
@@ -86,6 +96,13 @@ export const UserMenu: React.FC<UserMenuProps> = (props) => {
                             iconColor={iconColor || menuItem.iconColor}
                             fontColor={fontColor || menuItem.fontColor}
                             backgroundColor={backgroundColor || menuItem.backgroundColor}
+                            styles={{
+                                title: {
+                                    fontSize: 16,
+                                    fontFamily: theme.fonts.regular.fontFamily,
+                                    fontWeight: theme.fonts.regular.fontWeight,
+                                },
+                            }}
```

Comment:
Colors look good, but adding this styles prop breaks the end-user style overrides functionality. You need to merge these style with any overrides that are being passed in through the menuItem, e.g.:

```
                {menuItems &&
                    menuItems.map((menuItem: InfoListItemProps, index: number) => {
                        const menuStyles = menuItem.styles || {};
                        return (
                            <InfoListItem
                                {...menuItem}
                                key={index}
                                onPress={(): void => {
                                    closeMenu();
                                    if (menuItem.onPress) menuItem.onPress();
                                }}
                                iconColor={iconColor || menuItem.iconColor}
                                fontColor={fontColor || menuItem.fontColor}
                                backgroundColor={backgroundColor || menuItem.backgroundColor}
                                styles={
                                    Object.assign(menuStyles, {
                                        title: Object.assign({
                                            fontSize: 16,
                                            fontFamily: theme.fonts.regular.fontFamily,
                                            fontWeight: theme.fonts.regular.fontWeight,
                                        }, menuStyles.title)
                                    })
                                }
                            />
                        )
                    })}
```